py3-rouge-score.yaml

```
@@ -0,0 +1,40 @@
+package:
+  name: py3-rouge-score
+  version: 0.1.2
+  epoch: 0
+  description: Pure python implementation of ROUGE-1.5.5
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - numpy
+      - py3-absl-py
+      - py3-nltk
+      - py3-six
+      - python-3
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - wolfi-base
+      - busybox
+      - build-base
+      - python-3
+      - py3-setuptools
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: c7d4da2683e68c9abf0135ef915d63a46643666f848e558a1b9f7ead17ff0f04
+      uri: https://files.pythonhosted.org/packages/e2/c5/9136736c37022a6ad27fea38f3111eb8f02fe75d067f9a985cc358653102/rouge_score-${{package.version}}.tar.gz
```

rather than use this fetch URL with a sha I think we can use 
```suggestion
      uri: https://files.pythonhosted.org/packages/source/r/rouge_score/rouge_score-${{package.version}}.tar.gz
```
fwiw if you update your melange this should be the default behaviour for melange convert now

---

TIL that thanks!